### DISCUSSION TITLE
Speaker embedding in STT

### DISCUSSION THREAD

**Discussion by orena1** [espnet/espnet#5972] [Q&A] [answered]
Hi,
I did not find a lot of work on adding speaker embedding of some-kind to the speech-to-text model, does ESPNET support these kind of runs? or do you know of any relevant paper/code that implement it?

**Comment by sw005320** [comment]
Hi @Emrys365 and @simpleoier, I think we have some examples for target-speaker ASR. Can you list them here?

**Reply by Emrys365** [reply]
I don't think we have implemented target-speaker ASR directly. But it is possible to cascade [target-speaker extraction](https://github.com/espnet/espnet/blob/master/espnet2/enh/espnet_model_tse.py) functions and normal ASR in https://github.com/espnet/espnet/blob/master/espnet2/tasks/enh_s2t.py.

**Comment by orena1** [comment]
Thanks a lot, do you know about any relevant papers  that I can try to get examples from? Thanks

**Reply by Emrys365** [reply, marked as answer]
Sorry for the late reply.
I'm not sure what kind of papers you were asking for. But there is a recent paper comparing the use of different speaker embeddings for target-speaker processing, including ASR: [Investigation of Speaker Representation for Target-Speaker Speech Processing](https://arxiv.org/abs/2410.11243).
